Cargo.toml

```diff
@@ -9,17 +9,23 @@ readme = "README.md"
 repository = "https://github.com/awslabs/coldsnap"
 keywords = ["AWS", "Amazon", "EBS", "snapshot"]
 
+[features]
+default = ["rusoto-native-tls"]
+rusoto = ["rusoto-rustls"]
```

Comment:
Is this "rusoto" feature just for convenience? I'd tend to drop it unless it does something important.

Reply:
Yeah, copy-and-pasted from another project where it also probably should be there. Good call, removed.